c_plus_plus/src/view/views/local_conference_view.h

```diff
@@ -26,10 +28,12 @@ public slots:
     void hideEvent(QHideEvent *event) override;
 
 private:
-    QCameraInfo getCameraInfo();
+    QString getCameraDevice();
+    QString getOutputPath();
 
     Ui::LocalConferenceView *m_ui;
-    QCamera *m_camera;
+    Model::Recorder *m_recorder;
+    Model::ISettings *m_settings;
```

Comment:
There is no need for a pointer, this could be a reference I think.

Reply:
Patrick and Mathieu told me it's a better practice to use a pointer for a class member. We can discuss it later.